mypy/typeshed/stdlib/builtins.pyi

```diff
@@ -457,7 +457,7 @@ class str(Sequence[str]):
     def find(self, __sub: str, __start: SupportsIndex | None = ..., __end: SupportsIndex | None = ...) -> int: ...
     def format(self, *args: object, **kwargs: object) -> str: ...
     def format_map(self, map: _FormatMapMapping) -> str: ...
-    def index(self, __sub: str, __start: SupportsIndex | None = ..., __end: SupportsIndex | None = ...) -> int: ...
+    def index(self, __sub: str, __start: SupportsIndex | None = ..., __end: SupportsIndex | None = ...) -> int: ...  # type: ignore[override]
```

Comment:
whats the issue here, these are all positional right so why would it complain

Reply:
```py
class Sequence:
    def index(self, value: Any, start: int = 0, stop: int = ...) -> int: ...
class str(Sequence):
        def index(self, __sub: str, __start: SupportsIndex | None = ..., __end: SupportsIndex | None = ...) -> int: ...
```
It has a real name in the super type.